modules/installation-guide/partials/proc_importing-untrusted-tls-certificates-old.adoc

```diff
@@ -51,6 +51,13 @@ $ {orch-cli} create configmap ca-certs --from-file=__<certificate-file-path>__ -
 +
 To apply more than one certificate, add another `--from-file=_<certificate-file-path>_` option to the above command.
```

Comment:
It looks like your https://raw.githubusercontent.com/mmorhun/che-docs/che-20647/modules/installation-guide/partials/proc_importing-untrusted-tls-certificates-old.adoc file is not displayed in the preview of this PR.
It seems the preview of this PR shows content from https://raw.githubusercontent.com/mmorhun/che-docs/che-20647/modules/installation-guide/partials/proc_importing-untrusted-tls-certificates.adoc